articles/azure-vmware/tutorial-create-private-cloud.md

```diff
@@ -8,7 +8,7 @@ ms.date: 09/29/2021
 
 # Tutorial: Deploy an Azure VMware Solution private cloud
 
-The Azure VMware Solution private gives you the ability to deploy a vSphere cluster in Azure. For each private cloud created, there's one vSAN cluster by default. You can add, delete, and scale clusters.  The minimum number of hosts per cluster is three. More hosts can be added one at a time, up to a maximum of 16 hosts per cluster. The maximum number of clusters per private cloud is four.  The initial deployment of Azure VMware Solution has three hosts. 
+The Azure VMware Solution private gives you the ability to deploy a vSphere cluster in Azure. For each private cloud created, there's one vSAN cluster by default. You can add, delete, and scale clusters.  The minimum number of hosts per cluster is three. More hosts can be added one at a time, up to a maximum of 16 hosts per cluster. The maximum number of clusters per private cloud is twelve.  The initial deployment of Azure VMware Solution has three hosts. 
```

Comment:
@shaunjacob since the new number is over 10, we don't need to spell it out. Please replace 'twelve' with '12', thanks.

Reply:
@shaunjacob please change the spelled-out word 'twelve' to 12 so I can sign off on this update. Thanks